DICOMPlugins/DICOMM3DPlugin.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import absolute_import
+import glob
+import os
+import json
+import vtk
+import io
+import shutil
+import vtkSegmentationCorePython as vtkSegmentationCore
+import logging
+
+from base.DICOMM3DPluginBase import DICOMM3DPluginBase
+
+import slicer
+from DICOMLib import DICOMLoadable
+
+from SlicerDevelopmentToolboxUtils.mixins import ModuleLogicMixin
+from SlicerDevelopmentToolboxUtils.constants import DICOMTAGS
+from six.moves import range
+
+
+#
+# This is the plugin to handle translation of DICOM SEG objects
+#
+class DICOMM3DPluginClass(DICOMM3DPluginBase):
+
+  def __init__(self):
+    super(DICOMM3DPluginClass,self).__init__()
+    self.loadType = "DICOMM3D"
+
+  def examineFiles(self,files):
+    """ Returns a list of DICOMLoadable instances
+    corresponding to ways of interpreting the
+    files parameter.
+    """
+    loadables = []
+
+    # just read the modality type; need to go to reporting logic, since DCMTK
+    #   is not wrapped ...
```

Comment:
Try to make comments self-contained so that they will be meaningful to other developers in the future.  In this case, if the comment is needed at all, include some extra context about why the wrapping of DCMTK is important to mention here.